miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleNumberPerProjectService.java

```diff
@@ -38,8 +39,8 @@ public void setSampleNumberPerProjectDao(SampleNumberPerProjectDao sampleNumberP
     this.sampleNumberPerProjectDao = sampleNumberPerProjectDao;
   }
 
-  public void setSqlProjectDAO(SQLProjectDAO sqlProjectDAO) {
-    this.sqlProjectDAO = sqlProjectDAO;
+  public void setProjectStore(HibernateProjectDao sqlProjectDAO) {
```

Comment:
Parameter type (and name, less importantly) is still implementation instead of interface